flink-core/pom.xml

```diff
@@ -33,6 +33,25 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<surefire.module.config> <!--
+			required by JmxServer
+			-->--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED <!--
+			PluginConfigTest
+			-->--add-opens=java.base/java.util=ALL-UNNAMED <!--
```

Comment:
Is this necessary because of the `import java.util.Map`? That would be odd because we would probably have to add it to almost any module. :thinking:

Reply:
well, kinda.

add-open allows reflection against the given class.
add-exports is for getting access to JDK-internal classes, and implicitly does add-open.